internal/osutil/file_test.go

```diff
@@ -170,25 +187,10 @@ func TestCopyToDir(t *testing.T) {
 	})
 
 	t.Run("dest directory permission error", func(t *testing.T) {
-		tempDir := t.TempDir()
-		destTempDir := t.TempDir()
-		data := []byte("data")
-		// prepare file
-		filename := filepath.Join(tempDir, "a", "file.txt")
-		if err := WriteFile(filename, data); err != nil {
-			t.Fatal(err)
-		}
-		// forbid dest directory operation
-		if err := os.Chmod(destTempDir, 0000); err != nil {
-			t.Fatal(err)
+		if runtime.GOOS == "windows" {
+			t.Skip("skipping test on Windows")
 		}
-		defer os.Chmod(destTempDir, 0700)
-		if _, err := CopyToDir(filename, filepath.Join(destTempDir, "a")); err == nil {
-			t.Fatal("should have error")
-		}
-	})
```

Comment:
Is there any reason that this test case is removed?

Reply:
It looks the same, but I finally find out they are different. Added back the test case. Thanks!